aopy/preproc/quality.py

```diff
@@ -35,12 +37,77 @@ def bad_channel_detection(data, srate, lf_c=100., sg_win_t=8., sg_over_t=4., sg_
 
     psd_var = np.var(Sxx_low_psd,axis=0)
     norm_psd_var = psd_var/npla.norm(psd_var)
-    low_var_θ = np.mean(norm_psd_var)/3
+    low_var_θ = np.mean(norm_psd_var)/num_th
     bad_ch_mask = norm_psd_var <= low_var_θ
 
     return bad_ch_mask
```

Comment:
this is actually a good channel mask, right? can we change it to be channels with high variance instead to be consistent with the new function